docs/Changelog.md

```diff
@@ -14292,6 +14292,69 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="MeanSquaredDistance-12"></a>**MeanSquaredDistance-12**</a>
+
+  Loss function that measures the
+  mean squared distance (squared L2 norm) between each element in the 'scores'
+  and 'labels'.
+  
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), l_n = (score_n - label_n)^2
+  , N is the batch size.
```

Comment:
Line 14301-14305 can be shortened to

This operator first computes a element-wise squared loss tensor `L` via
```
L = Pow(Sub(score, label), 2)
```

Reply:
I think that makes it less readable